demo/package.json

```diff
@@ -15,6 +15,7 @@
     "gatsby-plugin-image": "1.1.0",
     "gatsby-plugin-mdx": "2.1.0",
     "gatsby-plugin-postcss": "4.1.0",
+    "gatsby-plugin-sharp": "^3.1.2",
```

Comment:
We typically install with `-E` to pin the exact version. This is how Renovate is configured in this repo - so doing it here will mean no further PR is opened automatically.